getting-started/bookshelf/config.go

```diff
@@ -37,84 +41,99 @@ var (
 	_ mgo.Session
 )
 
-const PubsubTopicID = "fill-book-details"
-
 func init() {
 	var err error
 
+	// Read config.json
+	configFile, err := ioutil.ReadFile("config.json")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Parse config.json
+	var configJson map[string]interface{}
```

Comment:
s/configJson/config/g

Also, initialisms in Go should have consistent case. So, this would be `configJSON`. https://github.com/golang/go/wiki/CodeReviewComments#initialisms